lms/envs/aws.py

```diff
@@ -500,3 +500,7 @@
 PDF_RECEIPT_COBRAND_LOGO_HEIGHT_MM = ENV_TOKENS.get(
     'PDF_RECEIPT_COBRAND_LOGO_HEIGHT_MM', PDF_RECEIPT_COBRAND_LOGO_HEIGHT_MM
 )
+
+if FEATURES.get('ENABLE_COURSEWARE_SEARCH'):
+    # Use ElasticSearch as the search engine herein
+    SEARCH_ENGINE = "search.elastic.ElasticSearchEngine"
```

Comment:
@jimabramson How was Elastic search configured in Student Notes? I remember there being some issue with making it the default. Should all our search APIs use a shared configuration somehow?